frontend/src/metabase/core/components/Icon/icons/index.ts

```diff
@@ -335,6 +335,8 @@ import variable_component from "./variable.svg?component";
 import variable_source from "./variable.svg?source";
 import verified_component from "./verified.svg?component";
 import verified_source from "./verified.svg?source";
+import verified_filled_component from "./verified_filled.svg?component";
+import verified_filled_source from "./verified_filled.svg?source";
```

Comment:
Should it use `fill="currentColor"` ?

Reply:
Yeah, should be fixed now to match the other `verified.svg`